### ISSUE TITLE
Candidaturas Salvas

### ISSUE THREAD

**Issue by Giresende** [SouJunior/products#476]

**Comment by Giresende** [comment]
Definição: Como candidato, eu gostaria fazer uma acompanhamento das vagas que me interessei, mas não não candidatei e salvei para visualizar mais tarde.

Descrição: O usuário ao acessar a sua conta, visualizará através do menu principal acima da a opção CANDIDATURA/VAGAS, a opção VAGAS SALVAS. 

Nesta página VAGAS SALVAS, teremos as vagas que o candidato se interessou, mas não chegou a se inscrever, apenas salvou para visualizar mais tarde.

![Image](https://github.com/SouJunior/products/assets/157187668/ad08ef4b-e5bf-4c15-bb8f-a6a249664b44)

_________________________________________________________________________________________________________________________________________

Critérios de Aceite:
Para que isso aconteça, precisamos inserir alguns botões funcionais no site.



_________________________________________________________________________________________________________________________________________
DoR - Definition of Ready

[  ] Ter pelo menos 3 critérios de aceite.
[  ] Conter regras de negócios e requisitos.
[  ] Contexto que comprovem a necessidade/viabilidade de construção da feature.
[  ] User story com esforço máximo de 21.
[  ] User Story validada pelo mentor.

 ________________________________________________________________________________________________________________________________________

DoD - Definition of Done

[  ] Documentação de usabilidade do incremento.
[  ] Evidência de teste do incremento.
[  ] Critérios de aceite atendidos.
[  ] PR aprovado em desenvolvimento.

**Comment by adrianoplaurindo** [comment]
@Giresende boa parte das considerações de #473 valem aqui tbm. :)

**Comment by adrianoplaurindo** [comment]
@Giresende pequenas considerações:
- Mudar o nome da US, está como Candidaturas, mas deveria ser Vagas;
- Prazo de exclusão da vaga: esses 30 dias foi baseado em alguma informação ou apenas definimos que é 30? E caso a vaga sexta excluída/fechada, qual o comportamento? Vamos apenas excluir? mostrar que a vaga foi encerrada? (aqui tu pode entender melhor com o time, o que acontece quando uma vaga é preenchida, se é excluída da base ou muda o status dela, por exemplo)

Realizando esses ajustes, vale compartilhar com o time de UX/Dev para colher feedbacks e levantamento de possíveis dúvidas :D

**Comment by Giresende** [comment]
Certo, vou analisar com o time essas questões e inserir os dados relevantes
na US.
Obrigada novamente.

Em qua., 3 de abr. de 2024 às 14:17, Adriano Laurindo <
***@***.***> escreveu:

**Comment by GabrielOrtunho** [comment]
https://www.figma.com/design/e6ULjE20hGthvbpoLPq2qZ/Portal-de-Vagas?node-id=3647-2565&t=jqR2dKoTMO3snByv-1

**Comment by Gisellepessanha** [comment]
@GabrielOrtunho Obrigada pela entrega. Levarei essa US para a próxima Planning.

**Comment by AlineBoechat** [comment]
@AlicePetry @Gisellepessanha podem me passar um Feedback sobre essa US? Pergunto para entender se faria sentido na gestão de acompanhamento

**Comment by Gisellepessanha** [comment]
Oi, @AlineBoechat !!
Acabamos de colocar essa US em sprint. 
Essa US trata das vagas em que o candidato se interessa, mas ainda não se candidatou, como forma de gerir a vagas de interesse. Após 7 dias essa vaga não permanecerá salva.
Não se isso responde sua pergunta. Caso não, é só me avisar, que responderei.

**Comment by biandradee** [comment]
Feita a integração da página de vagas salvas utilizando o FakeAPI como facilitador, enquanto os serviços não ficam prontos.

**Comment by bucardoso** [comment]
Aguardando retorno de devops
